### PR TITLE
Improve introduction

### PR DESCRIPTION
**Pull request by tcharding**
Attempt to improve the introduction page.

### CODE REVIEW

**Comment by tcharding** [comment]
This PR is made up of changes in https://github.com/rust-bitcoin/www.rust-bitcoin.org/pull/18, migrated here after implementing the review suggestions.

This PR can serve as a test run to see if the github action to deploy the site works. If the site is correctly deployed after this merges we can archive https://github.com/rust-bitcoin/www.rust-bitcoin.org (migrate issues first).

**Review by realeinherjar — APPROVED** [review]
You've migrated the suggestions! :)
ACK eaf026240ab19bb67094d3d3969024396b7e1f8b

File: cookbook/src/intro.md [hunk 1]
```diff
@@ -1,36 +1,25 @@
-# Rust Bitcoin
+# Welcome!
```
Comment (Kixunil) [inline cookbook/src/intro.md]: Titles should describe what the page is about - in this case `# Introduction to Rust Bitcoin` seems like a good option.

**Comment by realeinherjar** [comment]
And fix the typo in the PR Title as well

File: cookbook/src/intro.md (hunk 1 above)
Comment (tcharding) [reply cookbook/src/intro.md]: I agree with you in general but in this instance we have "Rust Bitcoin Cookbook" show right above in the "title bar" (not sure of term) so having "Introduction to Rust Bitcoin" and "introduction" in the sidebar seems a bit redundant and, totally subjectively, a bit dry. I don't really mind though. Thoughts?

Comment (Kixunil) [reply cookbook/src/intro.md]: Maybe just "Introduction" then. The thing on the left should match the title.

Comment (tcharding) [reply cookbook/src/intro.md]: Works for me!

**Comment by tcharding** [comment]
Changes in force push: `s/Welcome!/Introduction`

**Review by realeinherjar — APPROVED** [review]
ACK 85fb904a323a76eea8845c19294e2c22a2e5a9c6

**Comment by tcharding** [comment]
Will wait to use this PR as another test run of the gh-pages job.

**Comment by tcharding** [comment]
BOOM! Deployed successfully after merge. Thanks for your efforts @realeinherjar, this indeed turned out to be a better solution than my hacked up `build.sh` script - thanks for pushing me.

**Comment by realeinherjar** [comment]
Yeah no problem, sorry for the failed attempts. It was really hard to figure out what was set in the repo `Settings` without having access.
😄

**Comment by tcharding** [comment]
> It was really hard to figure out what was set in the repo Settings without having access.

Yep I have the same problem :)

**Pull request merged**